### DISCUSSION TITLE
Compatible gift product module

### DISCUSSION THREAD

**Discussion by arunguru77** [breezefront/community#120] [3. Integrations]
Hi support,
we are using breeze theme in our project , is there any module for gift type products/gift cards.
or suggest how can we make compatible easly.

Kind regards
Arun

**Comment by vovayatsyuk** [comment]
I don't know any to recommend. But the main rule - the less javascript has the module, the better for Breeze and store performance. 

Have you used something before that you are happy with? I can look at their demo and check how many scripts they use.

**Reply by arunguru77** [reply]
We used the following with Luma base theme, and it worked perfectly. 

https://amasty.com/gift-card-for-magento-2.html

**Reply by vovayatsyuk** [reply]
This one is hard to integrate, you have to be a js/magento frontend expert to make it work with breeze.

**Comment by dimdimich** [comment]
Please try to contact Amasty and inform them that you are using Breeze and interested in their module. The more people ask them, the greater the chances that they will add integration from their side. You can send them this link: https://packagist.org/packages/swissup/module-breeze/stats. We have already had discussions with them, and they said that if several people ask, they will add it from their side.
